appveyor.yml

```diff
@@ -92,13 +92,17 @@ matrix:
 #---------------------------------#
 #       build configuration       #
 #---------------------------------#
-
-
-install:
-  # Dump appveyor build vars for diagnostics
-  - "ECHO APPVEYOR_FORCED_BUILD: %APPVEYOR_FORCED_BUILD%"
-  - "ECHO APPVEYOR_RE_BUILD: %APPVEYOR_RE_BUILD%"
-
+before_build:
+#  - ps: gitversion /l console /output buildserver
+
+install: 
+  # using GitVersion
+  - cd %HTM_CORE%
+  - git fetch --unshallow
+  - GitVersion.exe
+  - GitVersion.exe /output buildserver
```

Comment:
where does the exe come from?

Reply:
It turns out that GitVersion.exe is installed in appveyor by default.
(I spent days trying to find something to tell me how to install it on appveyor... all for not)

Reply:
would there be a platfor/CI-provider agnostic solution? Because we'd want the same running on Travis/Circle/AppVeyor CIs